app/uk/gov/hmrc/apipublisher/services/PublisherService.scala

```diff
@@ -83,16 +83,19 @@ class PublisherService @Inject()(apiDefinitionConnector: APIDefinitionConnector,
         successful(None)
     }
 
+    def sameScopes(serviceScopes: Seq[Scope], requestedScopes: Seq[Scope]): Boolean = {
+      serviceScopes.toSet == requestedScopes.toSet
+    }
+
     def scopesRemainUnchanged(scopes: JsValue): Future[Option[JsValue]] = {
       val scopeSeq: Seq[Scope] = scopes.as[Seq[Scope]]
       val scopesSearch: immutable.Seq[String] = scopeSeq.map(s => s.key).toList
       val scopeServiceScopes: Future[Seq[Scope]] = apiScopeConnector.retrieveScopes(scopesSearch)
-
       scopeServiceScopes.map (serviceScopes => {
-        if (serviceScopes == scopeSeq) {
+        if(sameScopes(serviceScopes, scopeSeq)) {
           None
         } else {
-          Logger.error(s"scopes is $scopes,\nretrievedScopes is $serviceScopes")
+          Logger.error(s"requestedScopes is $scopeSeq,\nretrievedScopes is $serviceScopes")
```

Comment:
This is a bit misleading. "requestedScopes" looks like a variable name. Also the scopes in the publishing data aren't really _requested_ they're more being declared